src/pages/solutions/cloud/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React from "react";
+import Layout from "@theme/Layout";
+import styles from "./styles.module.css";
+import UseCaseRow from "@site/src/components/UseCaseRow";
+import Faq from "@site/src/components/Faq";
+import SimpleHeaderWithForm from "@site/src/components/SimpleHeaderWithForm";
+import PromotionalWaitingList from "@site/src/components/PromotionalWaitingList";
+import CloudStorageSvg from "@site/static/img/solutions/cloud/cloud_storage.drawio.svg";
+
+export default function CloudSolution(): JSX.Element {
+  return (
+    <Layout
+      title="Cloud Time Series Object Storage"
+      description="High-performance cloud storage for data of any size—images, text, sensor data, and more."
+    >
+      <main>
+        <SimpleHeaderWithForm
+          title="Cloud Time Series Object Storage"
```

Comment:
All well and good, but we're not using Azure for the SaaS. We build it on Google Cloud. :hugs: